apps/passport-server/src/services/generic-issuance/pipelines/instantiatePipeline.ts

```diff
@@ -29,30 +30,37 @@ export function instantiatePipeline(
     genericPretixAPI: IGenericPretixAPI;
   },
   zupassPublicKey: EdDSAPublicKey
-): Pipeline {
-  if (isLemonadePipelineDefinition(definition)) {
-    return new LemonadePipeline(
-      eddsaPrivateKey,
-      definition,
-      db,
-      apis.lemonadeAPI,
-      zupassPublicKey
-    );
-  } else if (isPretixPipelineDefinition(definition)) {
-    return new PretixPipeline(
-      eddsaPrivateKey,
-      definition,
-      db,
-      apis.genericPretixAPI,
-      zupassPublicKey
-    );
-  } else if (isCSVPipelineDefinition(definition)) {
-    return new CSVPipeline(eddsaPrivateKey, definition, db, zupassPublicKey);
-  }
+): Promise<Pipeline> {
+  return traced("", "instantiatePipeline", async (span) => {
```

Comment:
Is this first parameter meant to be an empty string?

Reply:
yes it was intended, but i can see how that may be confusing. i'll change it to something meaningful here and above + merge